ui/components/institutional/transaction-failed-modal/transaction-failed.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import withModalProps from '../../../helpers/higher-order-components/with-modal-props';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import Modal from '../../app/modal';
+import Box from '../../ui/box/box';
+import {
+  AlignItems,
+  BorderRadius,
+  DISPLAY,
+  FLEX_DIRECTION,
+  FONT_WEIGHT,
+  TEXT_ALIGN,
+  TextVariant,
+} from '../../../helpers/constants/design-system';
+import { Text } from '../../component-library';
+
+const TransactionFailedModal = ({
+  hideModal,
+  closeNotification,
+  operationFailed,
+  errorMessage,
+}) => {
+  const t = useI18nContext();
+  const handleSubmit = () => {
+    if (closeNotification) {
+      global.platform.closeCurrentWindow();
+    }
+    hideModal();
+  };
+
+  return (
+    <Modal onSubmit={handleSubmit} submitText={t('ok')}>
+      <Box
+        display={DISPLAY.FLEX}
+        flexDirection={FLEX_DIRECTION.COLUMN}
+        alignItems={AlignItems.center}
+        paddingLeft={4}
+        paddingRight={4}
+        style={{ flex: 1, overflowY: 'auto' }}
+      >
+        <img
+          className="transaction-failed__image"
+          src="images/icons/warning.svg"
+          alt=""
+        />
```

Comment:
I think we can replace this with the Warning icon component 

```
import { Icon, IconName } from '../../component-library';

.... 

<Icon name={IconName.Warning} size={IconSize.Xl} />
```

ref:
- https://metamask.github.io/metamask-storybook/?path=/story/components-componentlibrary-icon--default-story

Reply:
Fixed, thank you